neurodiffeq/solvers_utils.py

```diff
@@ -42,17 +43,34 @@ def load(cls, path):
                      t_max = t_max)
 					 
         return solver
-		
-		
 
-def save_solver(solver,filename):
+    #Saving the Solver Object
+    def save_solver(self,filename,path=pathlib.Path().absolute()):
+        PATH = str(path) + "\\" + filename

```

Comment:
It'd be better to use `os.path.join()` to nest path segments because linux and *nix use forward slash (`/`) and windows uses backward slash (`\`) to separate paths.